source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -34,6 +36,12 @@ void EXPECT_WARNING(const TResponse& response, const int expected_warning_id)
   EXPECT_EQ(expected_warning_id, response.status());
 }
 
+template <typename TResponse>
+void EXPECT_RFMX_ERROR(pb::int32 expected_error, const TResponse& response)
```

Comment:
nit can we match the EXPECT_)WARNING
flip the params so response is first and error is second
Rename expected_error to expected_error_id
Should just rename the method to EPECT_ERROR

Reply:
switched this around to more closely match